File: app/src/main/java/com/philsoft/metrotripper/app/MetroTripperApplication.java
package com.philsoft.metrotripper.app;

import android.support.multidex.MultiDexApplication;

import timber.log.Timber;

public class MetroTripperApplication extends MultiDexApplication {


    @Override
    public void onCreate() {
        super.onCreate();
        Timber.plant(new Timber.DebugTree());
    }
}

File: app/src/main/java/com/philsoft/metrotripper/app/MetroTripperApplication.kt
package com.philsoft.metrotripper.app

import android.support.multidex.MultiDexApplication

import timber.log.Timber

class MetroTripperApplication : MultiDexApplication() {

    override fun onCreate() {
        super.onCreate()
        Timber.plant(Timber.DebugTree())
    }
}
